Modules/EMCAL/src/ClusterTask.cxx

```diff
@@ -328,6 +328,10 @@ void ClusterTask::startOfCycle()
 
 void ClusterTask::monitorData(o2::framework::ProcessingContext& ctx)
 {
+  if (!o2::base::GeometryManager::isGeometryLoaded()) {
```

Comment:
by having a non-debug log in `monitorData` you will probably flood the infologger as soon as the geometry has not been loaded correctly...

Reply:
Done. Using static variable to display only once.